### ISSUE TITLE
Investigate cond over if chains

### ISSUE THREAD

**Issue by ewpatton** [mit-cml/appinventor-sources#949]
We should look into whether it is more efficient to map a if-elseif*-else block into a cond rather than a sequence of if-begin statements. For example, one outcome might be whether or not Kawa generates more efficient byte-code.

**Comment by halatmit** [comment]
Agreed, but i expect the differerences would be trivial given all the other
sources of slowdown.

== Hal

Hal Abelson
hal@mit.edu
Prof. of Comp. Sci. and Eng.
MIT Dept. of Elec. Eng. and Comp. Sci.
